### ISSUE TITLE
Extract values from shared matches used to make plot?

### ISSUE THREAD

**Issue by cheberling** [hms-dbmi/UpSetR#92]
I'll use the mutations example dataset to explain:

In the demo for the tool, a set of genes from the mutations dataset is plotted for the shared identifiers between the genes, which lies in column 1 of the dataset.  The plot gives the number of shared identifiers, but it doesn't tell what those identifiers are.  Is there a way to get the list of these identifiers used for plotting that fall into each matching group, and if not, could that be part of a future update?

**Comment by nickschurch** [comment]
I'll second that his would be a nice thing to be able to get from the data structure - which members belong to each part of the comparison,

**Comment by beacamara** [comment]
I completely agree with this suggestion. It would be very useful if we were able to get the different elements corresponding to each intersection category. I can see that in the upset web tool it is possible by selecting on the graph each category, but I can't see if this is possible with upset package in R.
Wonderful tool!
Thank you so much

**Comment by Maya0801** [comment]
Me too! any update on how to extract the list of shared identifiers?

**Comment by seb-mueller** [comment]
Just FYI, this has also been discussed in issue #85 and I ended up putting [something together](https://github.com/hms-dbmi/UpSetR/issues/85#issuecomment-415480954), looking forward for feedback if you end up trying it.

**Comment by Maya0801** [comment]
Works great!!! thanks a lot!

**Comment by kasadevall** [comment]
Same as seb-mueller, just posted a possible solution in the same thread...

> Just FYI, this has also been discussed in issue #85 and I ended up putting [something together](https://github.com/hms-dbmi/UpSetR/issues/85#issuecomment-415480954), looking forward for feedback if you end up trying it.
